src/App.tsx

```diff
@@ -1,61 +1,148 @@
 import './App.scss';
+import { ChangeEvent, FC, useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { User } from './types/User';
+import { Todo } from './types/Todo';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+function getUserById(id: number):User | null {
+  const foundUser = usersFromServer.find(user => user.id === id);
+
+  return foundUser || null;
+}
+
+function finderBiggestId(todos: Todo[]): number {
```

Comment:
The function name 'finderBiggestId' is not clear. Consider renaming it to 'findMaxTodoId' for better readability.